lib/node_loader.js

```diff
@@ -44,8 +44,6 @@ require('./credentials/environment_credentials');
 require('./credentials/file_system_credentials');
 require('./credentials/shared_ini_file_credentials');
 
-var SharedIniFile = require('./shared_ini');
```

Comment:
If we're exposing the IniLoader class, I think you'd want to require it in this file (assuming you're attaching the IniLoader to the AWS namespace to get docs showing). Right now the typings show that IniLoader is exported, but I don't think it actually is.